scrapy/loader/__init__.py

```diff
@@ -3,16 +3,26 @@
 See documentation in docs/topics/loaders.rst
 
 """
+from contextlib import suppress
```

Comment:
:lipstick: a-z sorting, both here and in the Scrapy imports below.

Reply:
Done, thanks!